ui/app/templates/vault/cluster/secrets/backend/overview.hbs

```diff
@@ -32,26 +32,31 @@
   <div class="box is-fullwidth is-shadowless has-tall-padding">
     <div class="selectable-card-container {{if (and (eq this.model.connections 403) (eq this.model.roles 403)) 'one-card'}}">
       {{#if this.model.connectionCapabilities.canList}}
-        <SelectableCard
+        <OverviewCard
+          data-test-selectable-card="Connections"
           @cardTitle="Connections"
-          @total={{if (eq this.model.connections 404) 0 this.model.connections.length}}
           @subText="The total number of connections to external databases that you have access to."
           @actionText="Configure new"
           @actionTo="vault.cluster.secrets.backend.create-root"
-          @queryParam={{"connection"}}
-          @type="role"
-        />
+          @actionQuery={{hash itemType="connection"}}
+        >
+          <Hds::Text::Display class="has-top-padding-m" @tag="h2" @size="500">
```

Comment:
Ahh thanks so much for updating this - love this so much better than using the `title-number` class! 💖